Traceback (most recent call last):
  File "showdown.py", line 146, in run
    result = self.play_game()
  File "showdown.py", line 131, in play_game
    self.selenium.move(move.move_index, move.backup_switch, mega=move.mega, volt_turn=move.volt_turn)
  File "/home/vasu/Work/pokemon_ai/browser.py", line 119, in move
    self.volt_turn(volt_turn)
  File "/home/vasu/Work/pokemon_ai/browser.py", line 175, in volt_turn
    self.volt_turn_switch(volt_turn)
  File "/home/vasu/Work/pokemon_ai/browser.py", line 167, in volt_turn_switch
    self.wait_for_move()
  File "/home/vasu/Work/pokemon_ai/browser.py", line 218, in wait_for_move
    close.click()
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webelement.py", line 60, in click
    self._execute(Command.CLICK_ELEMENT)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webelement.py", line 370, in _execute
    return self._parent.execute(command, params)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 173, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 164, in check_response
    raise exception_class(message, screen, stacktrace)
ElementNotVisibleException: Message: u'element not visible\n  (Session info: chrome=39.0.2171.95)\n  (Driver info: chromedriver=2.9.248304,platform=Linux 3.13.0-24-generic x86_64)' 
